<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="find_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>350</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="top">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(154, 93, 165);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="explain">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="note">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="show_lab">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 235, 235);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="explain_lab">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Èªë‰Ωì</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Ëß£Èáä</string>
   </property>
  </widget>
  <widget class="QLabel" name="note_lab">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Èªë‰Ωì</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Á¨îËÆ∞</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ÁºñËæë</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>üßô‚Äç</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>üîÖ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>show_lab</zorder>
  <zorder>find_edit</zorder>
  <zorder>top</zorder>
  <zorder>explain</zorder>
  <zorder>note</zorder>
  <zorder>explain_lab</zorder>
  <zorder>note_lab</zorder>
  <zorder>checkBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
